packages/streamline/src/components/cards/card-feedback/card-feedback.tsx

```diff
@@ -0,0 +1,57 @@
+import { Pressable } from 'react-native-ama';
+
+import { Box } from '../../../primitives/box/box';
+import { Icon } from '../../../primitives/icon/icon';
+import { Text } from '../../../primitives/text/text';
+import { IconsName } from '../../../types';
+import { Button } from '../../buttons/button/button';
+import { CardFeedbackProps } from './card-feedback.types';
+import { getCardFeedbackBackgroundColor } from './card-feedback.utils';
+import EmojiRating from './components/EmojiRating';
+import StarRating from './components/StarRating';
+import ThumbRating from './components/ThumbRating';
+
+const CardFeedbackContent = (props: CardFeedbackProps) => {
```

Comment:
Since you don't use props itself later on, it would be more consistent with the codebase to put directly
```tsx
({ currentRating, onRatingChange, type }: CardFeedbackProps)
```

Reply:
The CardFeedbackProps is an Union Type and `currentRating` is not available when `type === 'thumb'`. So it is not possible to destructuring like you propose